src/util/states.ts

```diff
@@ -427,19 +428,32 @@ function shouldSilent(el: Element, e: ElementEvent) {
 
 export function allLeaveBlur(api: ExtensionAPI) {
     const model = api.getModel();
+    const leaveBlurredSeries: SeriesModel[] = [];
+    const allComponents: ComponentModel[] = [];
     model.eachComponent(function (componentType, componentModel) {
         const componentStates = getComponentStates(componentModel);
+        const isSeries = componentType === 'series';
+        const view = isSeries ? api.getViewOfSeriesModel(componentModel as SeriesModel)
+            : api.getViewOfComponentModel(componentModel);
+        !isSeries && allComponents.push(componentModel);
+        isSeries && leaveBlurredSeries.push(componentModel as SeriesModel);
```

Comment:
Seems we only need to add `componentModel` to `leaveBlurredSeries` when `componentStates.isBlured`